vllm/utils.py

```diff
@@ -1524,9 +1524,15 @@ def __getitem__(self, key: Type[T]) -> _V:
         raise KeyError(key)
 
     def __contains__(self, key: object) -> bool:
+        return self.contains(key)
+
+    def contains(self, key: object, *, strict: bool = False) -> bool:
         if not isinstance(key, type):
             return False
 
+        if strict:
+            return key in self.data
```

Comment:
This fixes warnings being emitted when using Mantis model because both it and its base class (Llava) have objects in the registry.